CHANGELOG

```diff
@@ -1,4 +1,18 @@
 # Changelog
+## [0.2.9] - 2020-03-25
+### Bug Fixes
+ - Fix verification errors due to outdated values: https://github.com/nmstate/nmstate/issues/853
```

Comment:
Retry verification check when failed: https://github.com/nmstate/nmstate/issues/853

Reply:
I tried to make the fix more user-friendly.